crates/location_tracking_service/src/redis/commands.rs

```diff
@@ -175,26 +175,24 @@ pub async fn push_drainer_driver_location(
     city: &CityName,
     vehicle: &VehicleType,
     bucket: &u64,
-    geo_entries: &Vec<(Latitude, Longitude, DriverId)>,
+    geo_entries: &[(Latitude, Longitude, DriverId)],
 ) -> Result<(), AppError> {
     let geo_values: Vec<GeoValue> = geo_entries
-        .to_owned()
-        .into_iter()
+        .iter()
         .map(|(lat, lon, driver_id)| {
             prometheus::QUEUE_GUAGE.dec();
             GeoValue {
                 coordinates: GeoPosition {
-                    latitude: lat,
-                    longitude: lon,
+                    latitude: *lat,
+                    longitude: *lon,
```

Comment:
@SanchithHegde Why do we have to dereference it with `*` ? It gives same `f64` value only without `*` So why is it required and when exactly should we use it. Does it give performance benefits ?